categories.json

```
@@ -27,6 +27,16 @@
 				{"name": "game", "description": "Game servers"},
 				{"name": "web", "description": "Web servers"}
 			]},
+			{"name": "business", "description": "Business applications", "categories": [
+				{"name": "sap", "description": "SAP solutions"},
+				{"name": "salesforce", "description": "Salesforce solutions"},
+				{"name": "workday", "description": "Workday solutions"}
+			]},
+			{"name": "cloud", "description": "Cloud solutions", "categories": [
+				{"name": "azure", "description": "Azure cloud solutions"},
+				{"name": "aws", "description": "AWS Azure cloud solutions"},
```

you put a wrong description here (added Azure)

```suggestion
				{"name": "aws", "description": "Amazon Web Services cloud solutions"},
```